src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        ArrayList<Bulldozer> machines = new ArrayList<>();
+        machines.add(new Bulldozer());
+        machines.add(new Bulldozer());
+        machines.add(new Bulldozer());
```

Comment:
why not using a `for` loop with a predefined constant?

**should be applied to all other implementations**